domains/drbaph.json

```
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "Saganaki22",
+        "email": "info@drbaph.dev"
+    },
+    "record": {
+        "CNAME": "https://drbaph-srj.pages.dev/"
```

`CNAME` record must contain a domain name only.
```suggestion
        "CNAME": "drbaph-srj.pages.dev"
```